lxgettext/lxgettext.py

```diff
@@ -0,0 +1,190 @@
+import argparse
+import datetime
+import os
+import re
+
+import polib
+
+COLOUR_GREEN = '\033[92m'
+COLOUR_END = '\033[0m'
+
+KEYWORD = "gettext"
+gettext_re = re.compile("""%s\(['"](.+?)['"]\)""" % KEYWORD)
```

Comment:
@ema-fox me neither, but it is the simplest solution that does not require real parsing (and that is the problem as there is no all-in-one html, js, vue parser)